tools/build-jetpack.sh

```diff
@@ -61,9 +61,8 @@ hash yarn 2>/dev/null || {
     echo >&2 "Please install it following the instructions on https://yarnpkg.com. Aborting.";
     exit 1;
 }
-
-yarn --modules-folder=$TARGET_DIR/node_modules
-NODE_ENV=production BABEL_ENV=production $TARGET_DIR/node_modules/.bin/gulp
+yarn --cwd $TARGET_DIR cache clean
+yarn --cwd $TARGET_DIR run build-production
```

Comment:
This is cool, thank you! You can test if this works by going through testing instructions for #6308. Basically this script is needed if you want to have a built version of Jetpack clean from anything development related.